### DISCUSSION TITLE
the same as sgn-340.0

### DISCUSSION THREAD

**Discussion by isaak** [solgenomics/sgn#4365] [General]
-- Just testing github's tag release feature...
-- this is the same as sgn-340.0

<hr /><em>This discussion was created from the release <a href='https://github.com/solgenomics/sgn/releases/tag/sgn-342.1'>the same as sgn-340.0</a>.</em>

**Comment by lukasmueller** [comment]
Closing this discussion. If this is in error, please re-open.
